desc/configuration.py

```diff
@@ -1471,33 +1478,38 @@ def is_nested(self, nsurfs=10, ntheta=20, zeta=0, Nt=45, Nr=20):
             whether or not the surfaces are nested
 
         """
-        r_grid = LinearGrid(L=nsurfs, M=Nt, zeta=zeta, endpoint=True)
-        t_grid = LinearGrid(L=Nr, M=ntheta, zeta=zeta, endpoint=False)
-
-        r_coords = self.compute_toroidal_coords(r_grid)
-        t_coords = self.compute_toroidal_coords(t_grid)
-
-        v_nodes = t_grid.nodes
-        v_nodes[:, 1] = t_grid.nodes[:, 1] - t_coords["lambda"]
-        v_grid = Grid(v_nodes)
-        v_coords = self.compute_toroidal_coords(v_grid)
-
-        # rho contours
-        Rr = r_coords["R"].reshape((r_grid.L, r_grid.M, r_grid.N))[:, :, 0]
-        Zr = r_coords["Z"].reshape((r_grid.L, r_grid.M, r_grid.N))[:, :, 0]
-
-        # theta contours
-        Rv = v_coords["R"].reshape((t_grid.L, t_grid.M, t_grid.N))[:, :, 0]
-        Zv = v_coords["Z"].reshape((t_grid.L, t_grid.M, t_grid.N))[:, :, 0]
-
-        rline = MultiLineString(
-            [LineString(np.array([R, Z]).T) for R, Z in zip(Rr, Zr)]
-        )
-        vline = MultiLineString(
-            [LineString(np.array([R, Z]).T) for R, Z in zip(Rv.T, Zv.T)]
-        )
+        planes_nested_bools = []
+        if self._N == 0:
```

Comment:
see previous comment, its probably best not to override what the user gives as input (if for some reason they want to check 10 toroidal planes for an axisymmetric case, that's fine).

Also in general try to use `self.N` rather than `self._N` when reading values, that way any logic about what value to return will work correctly